### PR TITLE
[meta] update README

### PR DESCRIPTION
**Pull request by jrappen**
- fix links with `https`
- add reference section with links to docs

### CODE REVIEW

**Review by keith-hall — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -32,27 +36,38 @@ PS> git clone https://github.com/sublimehq/Packages.git
...
+ 2. Be as small and self-contained as is reasonable.
+    *Smaller changes are quicker to review.*
+ 3. Include a new, or enhanced, [syntax test](https://www.sublimetext.com/docs/syntax.html#testing) when changing a `.sublime-syntax` file.
  4. Have multiple syntax tests and a set of performance measurements (old vs. new) when making significant changes.
 
 ### New Packages
 
-Pull requests for new packages won't be accepted at this stage, as new packages can cause issues for users who have a package with the same name installed via Package Control. There are some planned changes that will address this in the future.
+Pull requests for new [packages](https://www.sublimetext.com/docs/packages.html) won't be accepted at this stage, as new packages can cause issues for users who have a package with the same name installed via [Package Control](https://packagecontrol.io/).
+There are some planned changes that will address this in the future.
```
Comment (deathaxe) [inline README.md:66]: Is this last sentence still valid?

File: README.md [hunk 2]
```diff
@@ -1,10 +1,14 @@
 # Sublime Packages
 
-> **These packages are developed against the latest [Sublime Text Dev Build](http://sublimetext.com/dev). Bugs may exist on older builds, and the format used is not compatible with builds older than 4095.**
+## Requirements
+
+These packages are developed against the latest Build of **Sublime Text** in the [`dev` channel](https://www.sublimetext.com/dev).
```
Comment (frou) [inline README.md:5]: Could also mention that the state of the repo for specific shipped ST builds can be seen by browsing tags: https://github.com/sublimehq/Packages/tags

**Comment by jrappen** [comment]
@michaelblyons sublimehq/Packages@97828e6 is fine with me.

File: README.md (hunk 1 above)
Comment (jrappen) [reply README.md:63]: I guess we can assume there won't be any feedback from SublimeHQ in the near future about this.

File: README.md (hunk 2 above)
Comment (jrappen) [reply README.md]: Addressed in sublimehq/Packages@bee3474.

**Review by keith-hall — APPROVED** [review]

**Review by michaelblyons — APPROVED** [review]

**Review by deathaxe — APPROVED** [review]

**Pull request merged**